lldb/packages/Python/lldbsuite/test/builders/builder.py

```diff
@@ -96,16 +98,119 @@ def getArchSpec(self, architecture):
         """
         return ["ARCH=" + architecture] if architecture else []
 
-    def getCCSpec(self, compiler):
+    def getToolchainSpec(self, compiler):
         """
-        Helper function to return the key-value string to specify the compiler
+        Helper function to return the key-value strings to specify the toolchain
         used for the make system.
         """
         cc = compiler if compiler else None
         if not cc and configuration.compiler:
             cc = configuration.compiler
+
         if cc:
```

Comment:
```suggestion
        if not cc:
          return []
```

.. and then unindent the rest of the code

Reply:
Done.